controllers/os-coreos/pkg/coreos/actuator_reconcile.go

```diff
@@ -61,6 +61,15 @@ func (c *actuator) cloudConfigFromOperatingSystemConfig(ctx context.Context, con
 				},
 			},
 		},
+		WriteFiles: []File{
```

Comment:
This file should only be written if `config.Spec.Purpose == extensionsv1alpha1. OperatingSystemConfigPurposeReconcile`.

Reply:
OK, updated PR